tailormap-components/projects/core/src/lib/user-interface/add-feature/add-feature.component.ts

```diff
@@ -57,6 +60,7 @@ export class AddFeatureComponent {
         this.visibleLayers.push(layerName);
       }
     });
+    this.changeDetection.detectChanges();
```

Comment:
@matthijsln Het knopje ging niet updaten als de lijst met lagen aangepast werd. Dit werkte, maar misschien heb jij een betere suggestie dan dit?

Reply:
Dit zal te maken hebben dat het event niet in de angular zone runt. Dit kan je checken met `NgZone.isInAngularZone()`

Reply:
Check, thanks :+1: